app/views/notifier/review_orders.html.haml

```diff
@@ -1 +1 @@
-= html "body_html", accounts: @accounts, login_url: transactions_in_review_url
+= html "body_html", accounts: @accounts, facility: @facility, login_url: transactions_in_review_url
```

Comment:
This is kind of picky, but I think the email will look better if each account is separated by a single line break rather than blank lines between them. I would consider splitting the localization into separate parts and render the accounts in between them.